www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/httpserver.xml

```diff
@@ -208,6 +215,13 @@
     <advanced>true</advanced>
     <help>Blocks files like .htaccess files or other files not intended for the public.</help>
   </field>
+  <field>
+    <id>httpserver.disable_gzip</id>
```

Comment:
not sure if this should be changed to disable_compression - just in case it is also important for other compressions such as `brotli` or `deflate`, which are both HTTP standard.

Reply:
I would use separate checkboxes to disable `brotli` (if there is such a request) so that there is a way to not disable compression entirely. The addition of the ability to disable `gzip` is due to that this is still used as a way to avoid a BREACH attacks.